mix.exs

```diff
@@ -36,10 +36,9 @@ defmodule Elixometer.Mixfile do
 
   defp deps do
     [
-        {:meck, "~> 0.8.3", only: :test},
-        {:edown, "~>0.7.0"},
-        {:lager, "~>3.0.2"},
-        {:exometer_core, "~>1.4.0"},
+        {:meck, "~> 0.8.3", override: true},
+        {:edown, "~>0.7.0", override: true},
+        {:exometer_core, github: "Feuerlabs/exometer_core"},
```

Comment:
We've had nothing but heartache and build failures using Feuerlabs's exometer. They have a very complex build stage. Have you been able to get this to build consistently?